identity-core/src/utils/base_encoding.rs

```diff
@@ -115,6 +115,22 @@ where
   bs58::encode(data).with_alphabet(bs58::Alphabet::BITCOIN).into_string()
 }
 
+/// Decodes the given `data` as base64.
+pub fn decode_b64<T>(data: &T) -> Result<Vec<u8>>
+where
+  T: AsRef<[u8]> + ?Sized,
+{
+  base64::decode_config(data.as_ref(), base64::URL_SAFE).map_err(Error::DecodeBase64)
+}
+
+/// Encodes the given `data` as base64.
+pub fn encode_b64<T>(data: &T) -> String
+where
+  T: AsRef<[u8]> + ?Sized,
+{
+  base64::encode_config(data.as_ref(), base64::URL_SAFE)
+}
+
```

Comment:
Ideally we should have some basic unit tests for these functions, like the other base encodings.

Have we settled on whether to use multibase instead?